.travis.yml

```diff
@@ -115,8 +108,7 @@ install:
   # for python 3 we dont install everything yet..
   - |
       [ $TESTS != py3 ] || \
-      pip install -e slave \
```

Comment:
so here, I'd guess we would need pip install buildbot-slave in order to not skip the compat tests